jenkins/_jenkins.py

```diff
@@ -171,9 +174,74 @@ def classify_build_log(log, path):
     return "Unknown"
 
 
+def get_week_number(timestamp):
+    """
+    Return a week number for the timestamp.
+
+    This combines the year and the week number so that
+    it can handle per-week processing of data that
+    spans multiple years.
+
+    :param datetime timestamp: The timestamp to convert
+        to a week number.
+    :return int: the week number.
+    """
+    dt = datetime.datetime.fromtimestamp(float(timestamp)/1000)
+    return dt.year * 100 + dt.isocalendar()[1]
+
+
+def get_numeric_result(result_str):
+    """
+    Convert a test result status in to a number.
+
+    :param str result_str: test result status
+        ("SUCCESS", "FAILED", etc.)
+    :return int: 100 if the test passed, 0 if it
+        failed. These mean of these values will
+        be the percentage of successful tests.
+    """
+    if result_str == SUCCESS:
+        return 100
+    return 0
```

Comment:
Indeed, but the search results for how to display a float as a percentage
showed how to override the global pandas float formatting. I guess an
extra map before display would probably work.